Description: Fixes a build failure on ppc
Origin: backport, http://trac.webkit.org/changeset/198919/webkit
Bug: https://bugs.webkit.org/show_bug.cgi?id=156015
--- a/modules/web/src/main/native/Source/JavaScriptCore/heap/MachineStackMarker.cpp
+++ b/modules/web/src/main/native/Source/JavaScriptCore/heap/MachineStackMarker.cpp
@@ -86,7 +86,11 @@
     }
 
     ucontext_t* userContext = static_cast<ucontext_t*>(ucontext);
+#if CPU(PPC)
+    thread->suspendedMachineContext = *userContext->uc_mcontext.uc_regs;
+#else
     thread->suspendedMachineContext = userContext->uc_mcontext;
+#endif
 
     // Allow suspend caller to see that this thread is suspended.
     // sem_post is async-signal-safe function. It means that we can call this from a signal handler.
